Emby.Naming/AudioBook/AudioBookFileInfo.cs

```diff
@@ -3,41 +3,41 @@
 namespace Emby.Naming.AudioBook
 {
     /// <summary>
-    /// Represents a single video file.
+    ///     Represents a single video file.
```

Comment:
These should not be indented, it should match the style of the rest of the project